package.json

```diff
@@ -106,7 +106,7 @@
         "ripemd160": "^2.0.2",
         "rxjs": "^6.6.7",
         "rxjs-compat": "^6.6.3",
-        "symbol-openapi-typescript-fetch-client": "^1.0.2",
+        "symbol-openapi-typescript-fetch-client": "^1.0.2-SNAPSHOT.202111091522",
```

Comment:
can we bump the version instead of adding SNAPSHOT tag?

Reply:
All dependencies have been released, using final versions